### DISCUSSION TITLE
[Copilot Labs] Infinite Loading on extension view

### DISCUSSION THREAD

**Discussion by loxator** [community/community#49068] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi there, I had signed up for Copilot labs last week and tried to take it for a spin. It worked(?) ok for the first time, but since then I am stuck in an infinite load loop for the 4 views . Am I not using it correct 🤔 ?

The below is what I see, regardless of highlighting a code snippet or not. I have also tried removing myself from the preview and signing up again, reinstalling the extension and disabling/enabling it again.

Any help is greatly appreciated!

https://user-images.githubusercontent.com/18169190/222680458-9fbc1787-a13e-4386-8ba4-efce90f5266d.mov

**Comment by mkdir700** [comment]
Yes, I have encountered the same problem as well.

I found through packet capture that this may be because of a failed login attempt from Copilot Labs.

```json
{
  "ver": 1,
  "sampleRate": 100,
  "tags": {
    "ai.application.ver": "unknown",
    "ai.device.id": "",
    "ai.cloud.roleInstance": "REDACTED",
    "ai.device.osVersion": "Darwin 22.2.0",
    "ai.cloud.role": "Web",
    "ai.device.osArchitecture": "arm64",
    "ai.device.osPlatform": "darwin",
    "ai.internal.sdkVersion": "node:unknown",
    "ai.user.id": "e4645d55960fdae33c6ed3dad3e11bfff7ff8960ba17b55eb6c70d5a64d8cbf2",
    "ai.session.id": "e07dc344-a4cd-49fc-8ac3-36fc144781ed1677945754628"
  },
  "data": {
    "baseType": "EventData",
    "baseData": {
      "ver": 2,
      "properties": {
        "github_copilot-labs_showBrushesLenses": "true",
        "github_copilot-labs_showTestGenerationLenses": "true",
        "copilot_build": "742",
        "copilot_buildType": "prod",
        "editor_version": "vscode/1.76.0",
        "editor_plugin_version": "copilot/0.11.742",
        "client_machineid": "e4645d55960fdae33c6ed3dad3e11bfff7ff8960ba17b55eb6c70d5a64d8cbf2",
        "client_sessionid": "e07dc344-a4cd-49fc-8ac3-36fc144781ed1677945754628",
        "copilot_version": "copilot/0.11.742",
        "common_extname": "copilot",
        "common_extversion": "0.11.742",
        "VSCode.ABExp.Features": "reusableLinks;mindaroBinariesVersion-1.0.20210702;mindaroBinariesVersion-1.0.20210723",
        "abexp.assignmentcontext": "vsreu685:30147344;bridge0708:30335490;bridge0723:30353136;",
        "common_os": "darwin",
        "common_platformversion": "22.2.0",
        "common_vscodemachineid": "e4645d55960fdae33c6ed3dad3e11bfff7ff8960ba17b55eb6c70d5a64d8cbf2",
        "common_vscodesessionid": "e07dc344-a4cd-49fc-8ac3-36fc144781ed1677945754628",
        "common_vscodeversion": "1.76.0",
        "common_isnewappinstall": "false",
        "common_uikind": "desktop",
        "common_remotename": "none"
      },
      "measurements": {
        "timeSinceIssuedMs": 10
      },
      "name": "copilot-labs/auth.github_login_failed"
    }
  },
  "iKey": "50ec6d81-d1d1-4f83-be10-0b6d513e2be0",
  "name": "Microsoft.ApplicationInsights.50ec6d81d1d14f83be100b6d513e2be0.Event",
  "time": "2023-03-04T16:03:13.341Z"
}
```

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @loxator ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by loxator** [comment]
@mkdir700 You were right, it was an issue with logging in but I had no idea about it as Github copilot was working ok. Seems like I had to log in again from the bottom right corner which has a bell icon to show the hidden notifications. Thanks for your help!

**Reply by junery115** [reply]
Thanks I found this quite helpful. I also had the infinite loading with no so pointers as to what to do, thanks.

**Comment by thiagocarvalhodev** [comment]
I'm having the same issue. I'll try the @loxator approach.

Edit: It didn't work. :(

**Comment by projetorevs** [comment]
It happened with me, and you need simply to accept the github lab terms and conditions:

1- Install github copilot labs.
2- Login in your github account (bottom left corner). In case you haven't github copilot, install and also accept it's terms.
3- Click on the bell (bottom right corner) to accept github lab terms. It will redirect you to their page to accept the conditions.

And it will be available for you.
